Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -264,7 +264,7 @@ public extension UIView {
 	/// SwifterSwift: Add array of subviews to view.
 	///
 	/// - Parameter subviews: array of subviews to add to self.
-	public func addSubviews(_ subviews: [UIView]) {
+	public func add(_ subviews: UIView...) {
```

Comment:
@Yoloabdo Thanks for taking the time to submit this PR 💯 I'm going to disagree that this is a better API. The name `addSubviews` builds on the `addSubview` API of UIKit. In my opinion, the name `add` is not descriptive enough.

 I'd also prefer to have the argument be typed as an array rather than having a variadic parameter.

Reply:
alright then, it's a matter of taste by the end I guess 👍

Reply:
Hey @Yoloabdo, thank you for your PR :)
It is actually not a matter of taste, our goal here at SwifterSwift is to keep naming conventions very close to Apple's.
- `add` is a very generic name for this case, you can add a subview, layout constraints, animations, gesture recognizers, and more to a view, that's why the extra `views` keyword adds a value here as @SD10 mentioned above.
- a variadic parameter accepts **zero or more** values of a UIView, users will be able to call the function without passing any arguments, like:
```swift
view.add()
```
which I think is not the desired usecase for this extension

Reply:
@omaralbeik Good addition on the behavior of a variadic 👍